src/main/resources/config.yml

```diff
@@ -20,6 +20,8 @@ allowedOrigins: ${CORS_ALLOWED_ORIGINS}
 logging:
   level: ${ROOT_LOG_LEVEL:-INFO}
   appenders:
+  - type: console
```

Comment:
not super familiar with this part of the config, could you help me understand why this was added for logging? Was there a threshold before?

Reply:
it's for tests. you can't override whole objects using dropwizard's `ConfigOverride` hooks, only single fields. i don't want json logs in the tests because it makes it harder to read when i'm troubleshooting failing tests. having a normal console reporter that defaults to an `OFF` threshold means it won't log anything in normal deployments, but i can override it in the tests to print normal human-readable logs.